bin/sfdx-lwc-jest

```diff
@@ -13,8 +13,16 @@ const { error } = require('../src/log');
 
 const args = getArgs();
 const runJest = require('../src/utils/test-runner');
-runJest(args);
 
-process.on('unhandledRejection', reason => {
+runJest(args)
```

Comment:
Delegating all the `process.exit` handling to the bin file instead of having this all around the code base.